lib/handlers/quote/quote.ts

```diff
@@ -222,6 +223,7 @@ export class QuoteHandler extends APIGLambdaHandler<
 
     const routingConfig: AlphaRouterConfig = {
       ...DEFAULT_ROUTING_CONFIG_BY_CHAIN(chainId),
+      ...{ distributionPercent: DISTRIBUTION_PERCENT_CONFIG(chainId, currencyIn, currencyOut, type, amountRaw) },
```

Comment:
this will throw an error for non-eth requests.

We should either return an empty object or the object with the distributionpercent.